core/language/en-GB/Help/server.tid

```diff
@@ -27,12 +27,14 @@ For example:
 --server 8080 $:/core/save/all text/plain text/html MyUserName passw0rd
 ```
 
-The username and password can be specified as empty strings if you need to set the hostname or pathprefix and don't want to require a password:
+The username and password can be specified as empty strings if you need to set the hostname or pathprefix and don't want to require a password. Note that you can use address 0.0.0.0 to open up your server to the entire local network::
 
 ```
---server 8080 $:/core/save/all text/plain text/html "" "" 192.168.0.245
+--server 8080 $:/core/save/all text/plain text/html "" "" 0.0.0.0
 ```
 
+Be aware that in public locations (coffee shops, libraries, airports, etc.), opening up your server can pose a security risk.
```

Comment:
IMO not `can pose`  ... It **does pose**

Reply:
I'm not sure that this is the best place to add this note. The tiddler $:/language/Help/server is what is displayed in response to `tiddlywiki --help server`, and so cannot have links to other entries. There's also $:/language/Help/listen for the listen command, to which it also applies.

In the prerelease, there's a new base for server documentation -- see https://tiddlywiki.com/prerelease/#WebServer

Perhaps we should (a) update the main web server documentation to explain the risks of 0.0.0.0 (e.g. we could do it [here](https://tiddlywiki.com/prerelease/#WebServer%20Parameter%3A%20host)) and (b) add a note to listen.tid and server.tid directing the user to the main web server docs for full details.